backend-tests/tests/test_rbac_v2.py

```diff
@@ -1508,3 +1511,601 @@ def test_get_emails_by_group(self, clean_mongo, test_case):
+                        ("description", (None, "description")),
+                        ("size", (None, str(os.path.getsize(artifact)))),
+                        (
+                            "artifact",
+                            (
+                                artifact,
+                                open(artifact, "rb"),
+                                "application/octet-stream",
+                            ),
+                        ),
+                    ),
+                )
+            assert r.status_code == 201
+
+        # get all artifacts
+        r = api_client.call("GET", deployments.URL_DEPLOYMENTS_ARTIFACTS)
+        assert r.status_code == 200
+        all_artifacts = r.json()
+        assert len(all_artifacts) == 2
+        foo_artifact_id = ""
+        bar_artifact_id = ""
+        for a in all_artifacts:
+            if a["name"] == "bar":
+                bar_artifact_id = a["id"]
+            elif a["name"] == "foo":
+                foo_artifact_id = a["id"]
+
+        # switch to user with limited access
+        test_user = create_user(tid=tenant.id, **test_case["user"])
+        login(test_user, test_case["use_personal_access_token"])
+        api_client.with_auth(test_user.token)
+        api_client_v2.with_auth(test_user.token)
+
+        r = api_client.call("GET", deployments.URL_DEPLOYMENTS_ARTIFACTS)
+        assert r.status_code == 200
+        artifacts = r.json()
+        assert len(artifacts) == 0
+
+        # tag the release
+        api_client_v2.with_auth(tenant.users[0].token)
+        release_url = deployments_v2.URL_RELEASE_TAGS.replace("{release_name}", "foo")
+        r = api_client_v2.call("PUT", release_url, test_case["tags"])
+        assert r.status_code == 204
+
+        # get artifacts after tagging
+        api_client.with_auth(test_user.token)
+        api_client_v2.with_auth(test_user.token)
+
+        r = api_client.call("GET", deployments.URL_DEPLOYMENTS_ARTIFACTS)
+        assert r.status_code == 200
+        artifacts = r.json()
+        assert len(artifacts) == test_case["number_of_allwed_artifacts"]
+        assert artifacts[0]["name"] == "foo"
+
+        # try to upload artifact
+        baz_artifact = {"artifact_name": "baz", "device_types": ["arm1"], "size": 256}
+        api_client.headers = {}  # avoid default Content-Type: application/json
+        api_client.with_auth(test_user.token)
+        with get_mender_artifact(**baz_artifact) as artifact:
+            r = api_client.call(
+                "POST",
+                deployments.URL_DEPLOYMENTS_ARTIFACTS,
+                files=(
+                    ("description", (None, "description")),
+                    ("size", (None, str(os.path.getsize(artifact)))),
+                    (
+                        "artifact",
+                        (artifact, open(artifact, "rb"), "application/octet-stream",),
+                    ),
+                ),
+            )
+        assert r.status_code == 403
```

Comment:
```suggestion
            assert r.status_code == 403
```
probably does not make any difference